client/me/help/help-contact/index.jsx

```diff
@@ -494,7 +503,20 @@ const HelpContact = React.createClass( {
 		// Hide the olark widget in the bottom right corner.
 		olarkActions.hideBox();
 
-		return <HelpContactForm { ...contactFormProps } />;
+		const shouldShowTicketRequestErrorNotice = ! showChatVariation && null !== ticketSupportRequestError;
+
+		return (
+			<div>
+				{ shouldShowTicketRequestErrorNotice &&
+					<Notice
+						status='is-error'
+						text={ this.translate( 'An error occurred while requesting for email support settings. Please try again later.' ) }
```

Comment:
I think your message is much well-put than mine, so I c&p it in f86599d3d5cdba9d7991c771770deab77f7a4475 :P